pkg/controller/secret-format.go

```diff
@@ -51,7 +47,6 @@ func (c *Controller) EnsureSecretType(ctx context.Context, node graph.Resource)
 	case res.resource.(*corev1.Secret).Type:
 		return nil
 	case corev1.SecretTypeOpaque:
-		// nolint:exhaustive
```

Comment:
Did it become exhaustive?

Reply:
it seems outer nolint is enough, anyway I add it back